lib/MessageReader.js

```diff
@@ -166,7 +169,28 @@ const buildType = (lines, customParsers) => {
       complexTypeName = name;
     } else if (name.indexOf("=") > -1 || splits.indexOf("=") > -1) {
       // constant type parsing
-      // constants values are not part of individual records
+      const matches = line.match(/(\S+)\s*=\s*(.*)\s*/);
+      let value = matches[2];
+      if (type !== "string") {
+        try {
+          value = JSON.parse(value.replace(/\s*#.*/g, ""));
+        } catch (error) {
+          // eslint-disable-next-line no-console
+          console.warn(`Error in this constant definition: ${line}`);
+          throw error;
+        }
+        if (type === "bool") value = !!value;
```

Comment:
how do we feel about `value = Boolean(value);`?

Reply:
Ah sure, slightly clearer. Will change.